frontend/src/app/main/ui/workspace/sidebar/options/menus/fill.cljs

```diff
@@ -75,7 +75,10 @@
       [:div.element-set
        [:div.element-set-title
         [:span label]
-        [:div.add-page {:on-click on-delete} i/minus]]
+        (when (not disable-remove?)
+         [:div.add-page {:on-click on-delete} i/minus]
+        )
```

Comment:
Move these closing parenthesis/brakets to the previous line

Reply:
Sure! :pray: